Gemfile

```diff
@@ -8,6 +8,7 @@ gem "i18n"
 if RUBY_VERSION < "2.0"
   gem "json", '~> 1.8'
   gem "coveralls", "0.8.13", :require => false
+  gem "term-ansicolor", "1.3.2" # A dependency of coveralls. The next version requires ruby >= 2.0.
```

Comment:
It seems to be the reason, but this didn't fix it. We gave up at this point since we don't have direct access to your CI server.